src/query/storage/m3/types.go

```diff
@@ -48,4 +49,18 @@ type Querier interface {
 		query *genericstorage.FetchQuery,
 		options *genericstorage.FetchOptions,
 	) (encoding.SeriesIterators, Cleanup, error)
+
+	SearchRaw(
```

Comment:
Hm, instead of FetchRaw and SearchRaw (which match the M3DB method names), maybe we should call these FetchCompressed and SearchCompressed?  In the future we may want to change the M3DB method names too, compressed is definitely a little more accurate than raw.

Reply:
Sounds good, touching enough of things around this area, I might as well